subsys/bluetooth/common/log.h

```diff
@@ -41,6 +41,12 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME, LOG_LEVEL);
 #define BT_WARN(fmt, ...) LOG_WRN(fmt, ##__VA_ARGS__)
 #define BT_INFO(fmt, ...) LOG_INF(fmt, ##__VA_ARGS__)
 
+#if defined(CONFIG_BT_DEBUG_ISO_DATA)
+#define BT_ISO_DATA_DBG(fmt, ...) BT_DBG(fmt, ##__VA_ARGS__)
+#else
+#define BT_ISO_DATA_DBG(fmt, ...)
+#endif /* CONFIG_BT_DEBUG_ISO_DATA */
+
```

Comment:
This file `subsys/bluetooth/common/log.h` is going away. Could we put this directly in `iso.c`?

For context, PR's related to removal of `bluetooth/common/log.h`.
https://github.com/zephyrproject-rtos/zephyr/pull/51562
https://github.com/zephyrproject-rtos/zephyr/pull/51595

Reply:
I am OK with that. @cvinayak Do you agree?

Reply:
Updated with this suggestion